Zeeguu Reader/Zeeguu Reader/AddFeedTableViewController.swift

```diff
@@ -76,13 +84,16 @@ class AddFeedTableViewController: UITableViewController {
 		return cell!
 	}
 	
+	func textFieldChanged(notification: NSNotification) {
+		self.navigationItem.rightBarButtonItem?.enabled = urlField.text?.characters.count > 0
```

Comment:
could be a bit smarter, and not enable before the user provides at least http:// 
or you could add that as an initial value in the text maybe? 
or ...